test/utils.dart

```diff
@@ -1,5 +1,22 @@
+// Copyright 2015 Workiva Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
 import 'dart:async';
 
-Future nextTick() {
-  return new Future.delayed(new Duration(milliseconds: 1));
+Future nextTick([int numTicks = 1]) async {
+  int i = 0;
+  while (i++ < numTicks) {
+    return new Future.delayed(new Duration(milliseconds: 1));
```

Comment:
does this work? won't this just return the first future instead of delaying further with a larger `numTick` count?